fireTS/models.py

```diff
@@ -193,6 +199,13 @@ def predict(self, X, y):
                                 yhat])[0:len(y)]
         return ypred
 
+    def forecast(self, X):
```

Comment:
forecast is not suitable for DirectAutoRegressor, because it is just fitting a function f for y(t+k) = f(y(t), y(t-1), ..., y(t-p), X(t), X(t-1), ..., X(t-q)). It cannot do a forecast for arbitrary step ahead. Please remove this method :)